app/_src/deck/guides/konnect.md

```diff
@@ -175,6 +175,18 @@ To target a specific geo, set `konnect-addr` to one of the following:
 
 {% endif_version %}
 
+{% if_version gte:1.40.x %}
```

Comment:
issue: Both this block and the one above render when version = 1.40.x. Either this block should be > 1.40.x or the one above should be < 1.40.x

![double-rendering](https://github.com/user-attachments/assets/15f2bce3-34d8-4aaf-8342-e8e8ecd79907)

Reply:
@kinman Thanks for catching that! I've fixed this issue in the latest commit